### DISCUSSION TITLE
6 Potential Projects - W2

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#86] [Week 2 - Discussions]
I am not sure my mind is aware of all of the potential possibilities for projects, so feel free to offer up alternate suggestions! 

* Create a game . . . I feel like this would be the most enjoyable project for me to work on!
* Build a restaurant website . . . I have a friend in need of a super simple site for their burger restaurant and this could be a super simple starting project
* Design an interactive poster for an upcoming music festival . . . This is a sort of vague idea, but maybe some sort of digital poster that doubles as a schedule/calendar when hovered over or clicked-on
* A personal website with different global times on it . . . just a bunch of clocks
* A piece of digital art . . . Could be cool and challenging, nft vibes
* A website archiving and cataloging different flowers native to Utah . . . I like flowers and am trying to xeriscape the front yard

**Comment by metasean** [comment]
All of these are great ideas!  I'm looking forward to seeing which one you settle on!
